pkg/util/tracing/tracer.go

```diff
@@ -608,7 +609,9 @@ func (t *Tracer) Extract(format interface{}, carrier interface{}) (*SpanMeta, er
 }
 
 // VisitSpans invokes the visitor with all active Spans.
-func (t *Tracer) VisitSpans(visitor func(*Span) error) error {
+func (t *Tracer) VisitSpans(
+	visitor func(*Span) error, // can return iterutil.StopIteration()
```

Comment:
I think it's a bit uncommon to place comments next to method arguments. This is better placed, with a full sentence, in the comment block explaining VisitSpans.